packages/react/src/components/accordion/Accordion.stories.tsx

```diff
@@ -23,6 +23,9 @@ export default {
 
```

Comment:
Could you fix a typo in the story arguments: buttton -> button?
https://github.com/City-of-Helsinki/helsinki-design-system/blob/7bb2fec51ab86ad54aff123792b14ee21899e846/packages/react/src/components/accordion/Accordion.stories.tsx#L19

Reply:
- [ ] Fix typo in the story arguments: buttton -> button